### DISCUSSION TITLE
Feature Request: Command and Permission to allow flight in claim area

### DISCUSSION THREAD

**Discussion by NickAwrist** [GriefPrevention/GriefPrevention#1836] [Ideas]
As a server owner, I want to use this plugin for my land claims and replace Towny, only thing stopping me is TownyFlight where players can reach a certain rank and have the ability to fly in their plots. Could this be something added that can be toggled in the config and with permissions?

**Comment by Jikoo** [comment]
GPFlags already has this functionality afaik.
